src/components/shifts/ShiftsTable.vue

```diff
@@ -198,91 +175,56 @@ export default {
     pastShifts: { type: Boolean, default: false }
   },
   emits: ["refresh"],
+  mixins: [ShiftUtilityMixin, breakpointsMixin],
+
   data: () => ({
-    icons: {
-      mdiCheck,
-      mdiClose,
-      mdiCircleMedium,
-      mdiTagOutline,
-      mdiFileDocumentOutline,
-      mdiPencil,
-      mdiSwapHorizontal,
-      mdiDelete
-    },
-    headers: SHIFT_TABLE_HEADERS,
-    colors: SHIFT_TYPE_COLORS,
-    typeIcons: SHIFT_TYPE_ICONS,
     selected: []
   }),
   computed: {
     flexHeaders() {
-      //check for tags and notes and hide column if none exist
-      let tagsAndNotes = 0;
-      this.shifts.forEach(
-        (shift) => (tagsAndNotes += shift.tags.length && shift.note.length)
+      const tagsAndNotesExist = this.shifts.some(
+        (shift) => shift.tags.length > 0 || shift.note.length > 0
       );
-      if (tagsAndNotes === 0) {
-        return this.headers.filter((item) => item.value !== "tagsNotes");
+
+      let filteredHeaders = [...this.headers];
+
+      if (this.isXs && this.isMobile) {
```

Comment:
As mentioned in the PR conversation: It would be great to have an indecator for the review status of a shift.